.github/ISSUE_TEMPLATE/other_issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Issue 
+about: Log an issue to improve SunPy!
```

Comment:
How is this different to a feature request (which is also a suggestion to improve sunpy)?

Reply:
For example, a suggestion to remove something from a sub-package is neither a feature request nor a bug fix. I'm sure we can think of others. I just wanted a catch-all so that no one ever uses the blank form.